README.md

```diff
@@ -624,47 +592,55 @@ br + br {
 
 <sup>[back to table of contents](#table-of-contents)</sup>
 
-
 ### Use `:empty` to Hide Empty HTML Elements
 
-If you have HTML elements that are empty, i.e., the content has yet to be set either by a CMS or dynamically injected (e.g., `<p class="error-message"></p>`) and it's creating unwanted space on your layout, use the `:empty` pseudo-class to hide the element on the layout. 
+If you have HTML elements that are empty, i.e., the content has yet to be set either by a CMS or dynamically injected (e.g., `<p class="error-message"></p>`) and it's creating unwanted space on your layout, use the `:empty` pseudo-class to hide the element on the layout.
 
 ```css
 :empty {
   display: none;
 }
 ```
 
+### Use `margin-inline` to add margin to both right and left of an html element,moreover when you want to set margin to top and bottom with same value use the shorthand `margin-block`
```

Comment:
lines 605 through 615 should be the only lines being included your pr is modifying a lot of lines please adjust this and resubmit

Reply:
cause of changes could be auto format setup in your IDE I'd start their